src/hooks/useSyncFocus/index.ts

```diff
@@ -0,0 +1,20 @@
+import {useLayoutEffect} from 'react';
+import type {RefObject} from 'react';
+import type {View} from 'react-native';
+
+/**
+ * Custom React hook created to handle sync of focus on an element when the user navigate through the app with keyboard.
+ * When the user navigate through an app using the arrows and then the tab button, the focus on the element and the native focus of the browser differs.
+ * To maintain consistency when an element is focused in the app, the focus() method is additionally called on the focused element to eliminate the difference between native browser focus and application focus.
+ */
+const useSyncFocus = (ref: RefObject<HTMLDivElement | View>, isFocused: boolean) => {
+    useLayoutEffect(() => {
+        if (!isFocused) {
+            return;
+        }
+
+        ref?.current?.focus();
```

Comment:
```suggestion
        ref.current?.focus();
```

I believe the ref itself is always there if the type is `RefObject<HTMLDivElement | View>`. I understand that bugs may appear in runtime, but the code should be written according to TS imo.